app/operations/tags/build_tags.rb

```diff
@@ -0,0 +1,14 @@
+module Tags
+  class BuildTags < Operation
+
+    array :tag_array
+
+    def execute
+      tags = tag_array.try(:map) do |tag|
```

Comment:
OOI - why `try` the map operation? If it's an empty array then it'll do nothing but it won't fail with missing method.

Reply:
Ah, because the guts of this method was originally plucked from attempting to map over (potentially omitted) params. Seems unnecessary.